.trivyignore

```diff
@@ -0,0 +1,2 @@
+# DoS vulnerability in otelhttp in scheduler
+CVE-2023-45142
```

Comment:
let's also add a short comment about why we add this exception here (i.e. the scheduler about to be removed soon)